diff --git a/bin.cjs b/bin.cjs
index 07274ea6f53fd54a129b3d338892feda6e995cff..2d255e0373b8ea91f7200e9158a3cfd23859f1ef 100644
--- a/bin.cjs
+++ b/bin.cjs
@@ -17858,6 +17858,7 @@ WHERE
             for (const checks of tableChecks) {
               let checkValue = checks.constraint_definition;
               const constraintName = checks.constraint_name;
+              if (checkValue == null) continue;
               checkValue = checkValue.replace(/^CHECK\s*\(\(/, "").replace(/\)\)\s*$/, "");
               checkConstraints[constraintName] = {
                 name: constraintName,
